Example/MoyaComparison/README.md

```diff
@@ -0,0 +1,10 @@
+# NetworkStackExample
+--------------------------
+
+## Coucou ##
```

Comment:
Good question. I think both should be in order:

* In the README at the root of the repo is supposed to describe the library, as a whole.
  * Take an example of good READMEs on my pods or others for example, they generally contains a quick description of the pod, install instructions, more detailed infos of features, and code examples or links to code examples.
  * So in this root README you'll place those descriptions and typically have a very short paragraph in the Examples section of that README telling the reader that there are 2 sample codes available in the `Examples/` folder, one for a general use case, one for Moya comparison
* This README inside the `MoyaComparison/` folder should in turn only describe the example project quickly, like:
  * telling that the aim of this example is to compare NetworkStack with Moya and to also compare some common JSON Parsing libraries while we're at it. It's the occasion to put links to the GitHubs of those other libs we're comparing
  * quickly describe how to quickly dive into the project and read it (like "the main VC is xxx…, to look at how Moya does xxx… you can look in the file xxx…, " and so on) so that a user wanting to explore the Example project avoid losing time figuring out how the project is structured
  * you might also have a paragraph explaining your (impartial, if possible) conclusions about the comparison. That could for example be a Markdown table to recap for each solution (Moya vs NetworkStack) which are the features supported by both, which are only supported by Moya, which only by NetworkStack… or some bullet points listing pros and cons of each solution